bindings/muen/start.c

```diff
@@ -58,11 +58,11 @@ DECLARE_ELF_INTERP
 /*
  * The "ABI1" Solo5 ELF note is declared in this module.
  *
- * Muen currently has no formal Solo5 ABI contract, so the version is always 1.
+ * Muen uses Solo5 ABI 2 as of commit ID .
```

Comment:
Can we formulate this as "Solo5/Muen uses ABI version 2 as of Muen commit <SHA>"?

Reply:
Done.